src/Parser/BrewmasterMonk/CombatLogParser.js

```diff
@@ -0,0 +1,66 @@
+// TODO:
+
+import React from 'react';
+
+import MainCombatLogParser from 'Parser/Core/CombatLogParser';
+
+import SuggestionsTab from 'Main/SuggestionsTab';
+import Tab from 'Main/Tab';
+import Talents from 'Main/Talents';
+
+// Core
+import HealingDone from './Modules/Core/HealingDone';
+import DamageTaken from './Modules/Core/DamageTaken';
+import DamageDone from './Modules/Core/DamageDone';
+import HealingReceived from './Modules/Core/HealingReceived';
+import Stagger from './Modules/Core/Stagger';
+import IronSkinBrew from './Modules/Spells/IronSkinBrew';
+
+// Features
+import CastEfficiency from './Modules/Features/CastEfficiency';
+import AlwaysBeCasting from './Modules/Features/AlwaysBeCasting';
+
+class CombatLogParser extends MainCombatLogParser {
+  static specModules = {
+    // Core
+    healingDone: HealingDone,
+    healingReceived: HealingReceived,
+    damageTaken: DamageTaken,
+    stagger: Stagger,
+    damageDone: DamageDone,
```

Comment:
I prefer that being in separate modules, I put the DamageTaken into its own module so that it didn't clutter the core combatlogparser with the objects of the damage breakdown. Feel free to change that to follow the model you decide on.